test/TestUtil.js

```diff
@@ -129,7 +129,7 @@ exports.markServerVersionAtLeast = function (_this, client, expectedVersion) {
     let actNumber;
     if (process.env['SERVER_VERSION']) {
         actNumber = BuildInfo.calculateServerVersionFromString(process.env['SERVER_VERSION']);
-    } else if (client != null) {
+    } else if (client !== null) {
```

Comment:
writing this comment for all changes in test/**/*.js

* all changes except == ----> ===, != ---> !== and line splitting due to long line are done automatically by eslint.